test/projective/affine.jl

```diff
@@ -96,10 +112,14 @@ include("../imports.jl")
             tfm = ScaleKeepAspect((32, 32))
 
             img = rand(RGB{N0f8}, 64, 96)
-            @test apply(tfm, Image(img)) |> itemdata |> size == (32, 48)
+            timg = apply(tfm, Image(img)) |> itemdata
+            @test timg |> size == (32, 48)
+            @test !any(isnan.(timg))
```

Comment:
Check bounds ranges here too:

```suggestion
            timg = apply(tfm, Image(img))
            @test getbounds(timg).rs == (1:32, 1:48)
            @test !any(isnan.(timg |> itemdata))
```